src/presenter.rs

```diff
@@ -348,10 +353,20 @@ impl<'a> Presenter<'a> {
             other => self.state = other,
         }
     }
+
+    fn suspend(&self, drawer: &mut TerminalDrawer<Stdout>) {
```

Comment:
This whole function should be guarded no? I would try to affect the windows behavior the least given it's the least tested platform

Reply:
Hmm, I'm not sure---if the whole function is guarded that means the method itself does not get compiled in right (so the function call above would fail)?

I don't use windoze so I can't test

Reply:
@mfontanini, if the function itself was guarded then you would also need to guard the function call since the function would not exist on windows:

```
error[E0599]: no method named `suspend` found for struct `Presenter` in the current scope
   --> src/presenter.rs:116:30
    |
43  | pub struct Presenter<'a> {
    | ------------------------ method `suspend` not found for this struct
...
116 |                         self.suspend(&mut drawer);
    |                              ^^^^^^^ method not found in `Presenter<'a>`
```

Reply:
Ah, that makes sense!